addons/website_sale/static/src/plugins/product_attribute_option_plugin.js

```diff
@@ -0,0 +1,45 @@
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+import { rpc } from "@web/core/network/rpc";
+
+class ProductAttributeOptionPlugin extends Plugin {
+    static id = "productAtttributeOption";
+    resources = {
+        builder_options: {
+            template: "website_sale.ProductAttributeOption",
+            selector: "#product_detail .o_wsale_product_attribute",
+            editableOnly: false,
```

Comment:
I don t know but you maybe need to add an access group ?

Reply:
There was no group in master. Just tested in master: a restricted editor is allowed to make this change.